packages/engine/Source/Scene/GaussianSplat3DTileContent.js

```diff
@@ -125,30 +125,27 @@ GaussianSplat3DTileContent.tilesetRequiresGaussianSplattingExt = function (
   tileset,
 ) {
   let hasGaussianSplatExtension = false;
-  let hasLegacyGaussianSplatExtension = false;
   if (tileset.isGltfExtensionRequired instanceof Function) {
     hasGaussianSplatExtension =
       tileset.isGltfExtensionRequired("KHR_gaussian_splatting") &&
       tileset.isGltfExtensionRequired(
         "KHR_gaussian_splatting_compression_spz_2",
       );
 
-    hasLegacyGaussianSplatExtension = tileset.isGltfExtensionRequired(
-      "KHR_spz_gaussian_splats_compression",
-    );
-  }
-
-  if (hasLegacyGaussianSplatExtension) {
-    deprecationWarning(
-      "KHR_spz_gaussian_splats_compression",
-      "Support for the original KHR_spz_gaussian_splats_compression extension has been deprecated in favor " +
-        "of the up to date KHR_gaussian_splatting and KHR_gaussian_splatting_compression_spz_2 extensions and will be " +
-        "removed in CesiumJS 1.135.\n\nPlease retile your tileset with the KHR_gaussian_splatting and " +
-        "KHR_gaussian_splatting_compression_spz_2 extensions.",
-    );
+    if (
+      tileset.isGltfExtensionRequired("KHR_spz_gaussian_splats_compression")
+    ) {
+      deprecationWarning(
+        "KHR_spz_gaussian_splats_compression",
+        "Support for the original KHR_spz_gaussian_splats_compression extension has been deprecated in favor " +
```

Comment:
I think the message should be "removed" rather than "deprecated".
`Support for the original KHR_spz_gaussian_splats_compression extension has been removed...`

Also should this be an error rather than a warning?

Reply:
@jjhembd Re throwing vs the warning I think this is ok for now. It looks like it gets caught further down the chain that it's an unsupported extension. Functionally it renders nothing in the viewer. I think throwing an error could cause it to fail/block rendering completely which I think is not desired. Happy if we want to reconsider in the future but I'm comfortable with this solution for now

<img width="1907" height="1449" alt="2025-11-03_12-45" src="https://github.com/user-attachments/assets/137b7f7e-d54b-4ed5-9197-f87669a8d087" />